client/src/components/CheckBox.js

```diff
@@ -4,19 +4,32 @@ export class CheckBox extends React.Component{
   render(){
     const {
       id,
-      onClick,
-      color,
-      label,
-      active,
-      isSolidBox,
-      container_style,
-      checkbox_style,
-      label_style,
-      checkmark_vertical_align,
-      disabled,
+      onClick, //required: id => {}
+      label, //required text for checkbox
+      active, //required, equivalent of 'checked'
+      color, //defaults to window.infobase_color_constants.primaryColor
+      isSolidBox, //defaults to false
+      container_style, //style for div, containing checkbox and label
+      checkbox_style, //style for checkbox
+      label_style, //style for label
+      a11y_label_style, //style for a11y label only
+      checkmark_vertical_align, //defaults to 0.1
+      disabled, //greys out checkbox and make it non-clickable if true
     } = this.props;
 
-    return <div style={{
+    return window.is_a11y_mode ?
+    <div className="checkbox">
+      <label style={a11y_label_style}>
+        <input 
+          type='checkbox'
+          checked={active}
+          onChange={onClick}
+          style={{marginRight: 5}}
+        />
+        { label }
+      </label>
+    </div>
```

Comment:
I'm confused about why you are applying a `style` at all to an a11y-specific version component....is there a particular reason for it?

Reply:
Do you mean `a11y_label_style`? It's because some text for checkbox are bolded for A11y version. However, this bold is not needed for standard version because it is `link-styled `(blue, underlined text)

If you meant `marginRight`, this is actually already applied to the new fancy checkbox with `legend-color-checkbox`

Reply:
`a11y_label_style` may be unnecessary though. I've only put it up to bold the text and I hated myself for doing it for adding another messy `style` props..

Reply:
I don't think the bold style is necessary.